package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hero-ui",
-  "version": "4.0.24",
+  "version": "4.1.0",
```

Comment:
I recommend leaving the version bump for a separate commit, this way it will trigger a deploy to S3 and NPM. The README has a little section on that now.